eng/pipelines/templates/jobs/build-conda-dependencies.yml

```
@@ -0,0 +1,130 @@
+parameters:
+  - name: CondaArtifacts
+    type: object
+    default: []
+
+jobs:
+- job: 'Build_Windows_Dependencies'
+  displayName: 'Build Windows Dependencies'
+  timeoutInMinutes: 90
+  pool:
+    name: azsdk-pool-mms-win-2022-general
+    vmImage: MMS2022
+  variables:
+    VS_INSTALLER_URL: "https://aka.ms/vs/17/release/vs_enterprise.exe"
+    VS_INSTALLER_PATH: "$(Build.BinariesDirectory)/vs_enterprise.exe"
+    VS_ENTERPRISE_PATH: "C:\\Program Files\\Microsoft Visual Studio\\2022\\Enterprise"
+  steps:
+    - bash: |
+        echo "##vso[task.prependpath]$(CONDA)/Scripts"
+      displayName: 'Prepend PATH with Conda'
+
+    - task: UsePythonVersion@0
+      displayName: 'Use Python $(PythonVersion)'
+      inputs:
+        versionSpec: $(PythonVersion)
+
+    - pwsh: |
+        pip install tools/azure-sdk-tools[build]
```

If you want any of these command failures to halt the pipeline:

```suggestion
    - pwsh: |
        $ErrorActionPreference = 'Stop'
        $PSNativeCommandUseErrorActionPreference = $true
        pip install tools/azure-sdk-tools[build]
```